diagnostic.md

```diff
@@ -36,19 +36,19 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+Relative path because it doesn't start at the root file
```

Comment:
Absolute because it in fact does start with the root directory `/Users/blah_blah/...` is the root.